Ticket-booking-spring-app/src/main/java/com/example/demo/entity/TicketPrice.java

```diff
@@ -0,0 +1,43 @@
+package com.example.demo.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+public class TicketPrice {
+    @Id
+    @Column(name="type", nullable = false)
+    private String type;
+    @Column(name="price", nullable = false)
+    private float price;
```

Comment:
Czy trzymanie wartości pieniężnych na flotach to dobry pomysł?

Reply:
Hmm, rzeczywiście floaty mają słabą precyzję więc może lepszy byłby BigDecimal.